components/esp32/cpu_start.c

```diff
@@ -237,9 +247,10 @@ void IRAM_ATTR call_start_cpu1()
 #endif
 
     wdt_reset_cpu1_info_enable();
-    ESP_EARLY_LOGI(TAG, "App cpu up.");
+    cpu1_stack = (uint32_t *) heap_caps_malloc(2048, MALLOC_CAP_8BIT);
```

Comment:
If you only need 8-bit memory then you should be able to call plain malloc() here

Reply:
i tried that t first, but for some reason if i change these to malloc/free i get a crash.